src/Goerli-DssSpell.sol

```diff
@@ -40,6 +34,14 @@ contract DssSpellAction is DssAction {
         return false;
     }
 
+    address constant internal MCD_CLIP_CALC_GUSD_A = 0x738EA932C2aFb1D8e47bebB7ed1c604399f2A99e;
+    address constant internal MCD_CLIP_CALC_USDC_A = 0x3a278aA4264AD66c5DEaAfbC1fCf6E43ceD47325;
+    address constant internal MCD_CLIP_CALC_PAXUSD_A = 0x8EE38002052CA938646F653831E9a6Af6Cc8BeBf;
+
+    address internal immutable STARKNET_GOV_RELAY = DssExecLib.getChangelogAddress("STARKNET_GOV_RELAY");
+    address internal immutable NEW_STARKNET_GOV_RELAY = 0x8919aefA417745F22c6af5AD6550E83159a373F3;
```

Comment:
solc version doesn't seem to match `0.7.6` as per [master](https://github.com/makerdao/starknet-dai-bridge/blob/main/contracts/l1/L1GovernanceRelay.sol#L16)

Reply:
where this would need to be checked against?

Reply:
https://github.com/makerdao/starknet-dai-bridge/tree/mk/govrelay-upgrade/contracts/l1
It will be merged after the deployment.

Reply:
ok, it matches the source provided.